lib/hawkular/metrics/metric_api.rb

```diff
@@ -36,7 +36,22 @@ def push_data(gauges: [], counters: [], availabilities: [])
       availabilities.each { |g| default_timestamp g[:data] }
 
       data = { gauges: gauges, counters: counters, availabilities: availabilities }
-      http_post('/metrics/data', data)
+      path = 'metrics/'
+      if new_api_supported
```

Comment:
Below you use the elvis operator ( ?: ) - please use one for consistency - I think in this case ?: is more appropriate